enterprise/reporting/mediator.go

```diff
@@ -12,16 +12,16 @@ import (
 	"github.com/rudderlabs/rudder-server/utils/types"
 )
 
-type ReportingMediator struct {
+type Mediator struct {
 	log logger.Logger
 
 	g         *errgroup.Group
 	ctx       context.Context
 	reporters []types.Reporting
```

Comment:
minor preference
```suggestion
	reporters []types.Reporter
```

Reply:
Is it ok if I handle this in a separate PR?